priv/site/_includes/tickets/marquee.slime

```diff
@@ -1,32 +1,9 @@
 marquee.marquee
-  a href="#"
-    span Buy Tickets Here
-    <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect width="56" height="56" fill="#E5E5E5"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M44.8681 27.6127L33.9229 14.7288L36.4419 12.5632L49.2268 27.6127L36.4419 42.6622L33.9229 40.4966L44.8681 27.6127Z" fill="black"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M6.56641 25.946L46.6983 25.946L46.6983 29.2794L6.56641 29.2794L6.56641 25.946Z" fill="black"/>
-    </svg>
-
-  a href="#"
-    span Buy Tickets Here
-    <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect width="56" height="56" fill="#E5E5E5"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M44.8681 27.6127L33.9229 14.7288L36.4419 12.5632L49.2268 27.6127L36.4419 42.6622L33.9229 40.4966L44.8681 27.6127Z" fill="black"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M6.56641 25.946L46.6983 25.946L46.6983 29.2794L6.56641 29.2794L6.56641 25.946Z" fill="black"/>
-    </svg>
-
-  a href="#"
-    span Buy Tickets Here
-    <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect width="56" height="56" fill="#E5E5E5"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M44.8681 27.6127L33.9229 14.7288L36.4419 12.5632L49.2268 27.6127L36.4419 42.6622L33.9229 40.4966L44.8681 27.6127Z" fill="black"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M6.56641 25.946L46.6983 25.946L46.6983 29.2794L6.56641 29.2794L6.56641 25.946Z" fill="black"/>
-    </svg>
-
-  a href="#"
-    span Buy Tickets Here
-    <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect width="56" height="56" fill="#E5E5E5"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M44.8681 27.6127L33.9229 14.7288L36.4419 12.5632L49.2268 27.6127L36.4419 42.6622L33.9229 40.4966L44.8681 27.6127Z" fill="black"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M6.56641 25.946L46.6983 25.946L46.6983 29.2794L6.56641 29.2794L6.56641 25.946Z" fill="black"/>
-    </svg>
+  = for x <- 1..4 do
+    a href="#"
```

Comment:
Oof, this actually should've been a `tito_url()`. Can you replace this with a call to `link/3`?

Reply:
I've changed all to `= link @env, to: tito_url() do`.

Let me know if I missed something–not sure what you meant with "Can you replace this with a call to `link/3`?"

Reply:
That's perfect, it was precisely it 👍 

`function/arity` is the notation to describe functions in Elixir. So `link/3` means "function named `link` that receives 3 arguments". Just a fancy way to distinguish between different functions and function branches.

Reply:
Ah, nice. Thank you.